modules/ROOT/pages/migration/index.adoc

```diff
@@ -16,14 +16,389 @@ To update your Neo4j GraphQL Library, use npm or the package manager of choice:
 npm update @neo4j/graphql
 ----
 
+== What's new
+
+Until version 5 of the Library, we tried to limit, as much as possible, the number of breaking changes and related migration pain. In the meantime, we introduced new features such as Aggregation, Fulltext search, Vector search, Connection API, and Interfaces&Union support, to name a few! This approach led to an inconsistent user-facing API with room for ambiguous behaviors.
```

Comment:
Do we need this explanation here? It feels a bit like an excuse.

Reply:
We can remove it!